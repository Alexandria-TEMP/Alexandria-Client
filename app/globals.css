@tailwind base;
@tailwind components;
@tailwind utilities;

@layer utilities {
  .text-balance {
    text-wrap: balance;
  }
}

@layer base {
  h1 {
    @apply text-2xl font-bold;
  }
  h2 {
    @apply text-xl font-semibold;
  }
  h3 {
    @apply text-lg;
  }
  h6 {
    @apply text-sm;
  }
}
