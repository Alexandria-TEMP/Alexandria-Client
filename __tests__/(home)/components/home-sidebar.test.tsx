import { render } from "@testing-library/react";
import { expect, describe, it } from "@jest/globals";
import HomeSidebar from "@/(home)/components/home-sidebar";

describe("HomeSidebar", () => {
  it("matches snapshot", () => {
    const { container } = render(<HomeSidebar />);
    expect(container).toMatchSnapshot();
  });
});
