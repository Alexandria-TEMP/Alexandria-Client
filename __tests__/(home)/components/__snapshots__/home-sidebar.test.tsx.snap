// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeSidebar matches snapshot 1`] = `
<div>
  <div
    class="sticky top-20"
  >
    <div
      class="shadow-black/5 shadow-none group relative overflow-hidden bg-content3 dark:bg-content2 before:opacity-100 before:absolute before:inset-0 before:-translate-x-full before:animate-[shimmer_2s_infinite] before:border-t before:border-content4/30 before:bg-gradient-to-r before:from-transparent before:via-content4 dark:before:via-default-700/10 before:to-transparent after:opacity-100 after:absolute after:inset-0 after:-z-10 after:bg-content3 dark:after:bg-content2 rounded-large"
      style="max-width: 300px;"
    >
       
      <img
        alt="Alexandria logo"
        class="relative z-10 shadow-black/5 data-[loaded=true]:opacity-100 shadow-none opacity-0 transition-transform-opacity motion-reduce:transition-none !duration-300 rounded-large"
        src="/logo/logo32white.svg"
        width="300"
      />
    </div>
    <div
      class="h-4"
    />
    <h1>
      Welcome to Alexandria!
    </h1>
    <div
      class="h-4"
    />
    <div
      class="text-center"
    >
      <span
        class="font-bold"
      >
        Alexandria
      </span>
       is an open-source platform for scientific collaboration. Its aim is to make science available for everyone and to promote spontaneous, international collaboration.
       
      <a
        class="text-primary"
        href="/about"
      >
        Read more about it.
      </a>
    </div>
    <div
      class="h-4"
    />
  </div>
</div>
`;
